vmdk_plugin/Makefile

```diff
@@ -334,6 +334,7 @@ checkremote:
 # expects binaries to be deployed ot the VM and ESX (see deploy-all target)
 test-vm: checkremote deploy-vm-test
 	$(log_target)
+	-$(SSH) root@$(ESX) '/etc/init.d/vmdk-opsd start'
```

Comment:
Due to issue #1291, we need to restart the service because it has crashed due to "Config Remove".